lib/logwriter.c

```diff
@@ -1176,16 +1170,11 @@ log_writer_realloc_line_buffer(LogWriter *self)
  *
  */
 
-static gboolean
+static LogProtoStatus
 log_writer_flush_finalize(LogWriter *self)
```

Comment:
`log_writer_flush_finalize()` is only used once, and as such, with this change the function becomes unnecessary. It might worth considering removing it.

Reply:
yes, but it has a "complex" logic, so it's worth having a dedicated function, also I do not want to split the code base even more